responses/cs/HassLightSet.yaml

```diff
@@ -0,0 +1,8 @@
+language: cs
+responses:
+  intents:
+    HassLightSet:
+      brightness: "Jas {{ slots.name }} nastaven na {{ slots.brightness }}"
+      brightness_area: "Jas {{ slots.area }} nastaven na {{ slots.brightness }}"
+      color: "Barva {{ slots.name }} nastavena na {{ slots.color }}"
+#      color_area: "Barva {{ slots.area }} nastavena na {{ slots.color }}"
```

Comment:
can we already add
Barva světel v ...

Reply:
Yes, there will be "Barva v <area>" in next commit. I have prepared it. But then migration come in place.

Good point with negative temp values. 
And I think, nobody will ever use "21 supeň" as it is gramaticly correct, but mosty used for ordinal numbers - 21. stupeň